### ISSUE TITLE
Wrong initialization for signed 24 bit integer (Int24s{b|l})

### ISSUE THREAD

**Issue by seyfullahari** [construct/construct#288]
Int24s{b|l}() (maybe also other function) does not construct BytesInteger() correctly. signed=True shall also be added during construction.

**Comment by arekbulski** [comment]
The constructors seem good. If you do not provide a clear test case that fails, I dont see what is wrong.
```py
@singletonfunction
def Int24ub():
    """A 3-byte big-endian unsigned integer, as used in ancient file formats."""
    return BytesInteger(3)
@singletonfunction
def Int24ul():
    """A 3-byte little-endian unsigned integer, as used in ancient file formats."""
    return BytesInteger(3, swapped=True)
@singletonfunction
def Int24sb():
    """A 3-byte big-endian signed integer, as used in ancient file formats."""
    return BytesInteger(3)
@singletonfunction
def Int24sl():
    """A 3-byte little-endian signed integer, as used in ancient file formats."""
    return BytesInteger(3, swapped=True)
```

**Comment by arekbulski** [comment]
Fixed in b7ba38f345a68b05cc1141a21a4bd4f47f735494 .
Big thanks for pointing to this huge bug! :cake:
